x-pack/plugins/ml/public/application/components/anomalies_table/anomaly_details.js

```diff
@@ -183,11 +184,54 @@ function getDetailsItems(anomaly, examples, filter) {
     });
   }
 
+  items.push({
+    title: (
+      <EuiToolTip
+        position="left"
+        content={i18n.translate('xpack.ml.anomaliesTable.anomalyDetails.recordScoreTooltip', {
+          defaultMessage:
+            'A normalized score between 0-100, which indicates the relative significance of the anomaly record result.',
+        })}
+      >
+        <span>
+          {i18n.translate('xpack.ml.anomaliesTable.anomalyDetails.recordScoreTitle', {
+            defaultMessage: 'Record score',
+          })}
+          <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+        </span>
+      </EuiToolTip>
+    ),
+    description: Math.floor(1000 * source.record_score) / 1000,
+  });
+
+  items.push({
+    title: (
+      <EuiToolTip
+        position="left"
+        content={i18n.translate(
+          'xpack.ml.anomaliesTable.anomalyDetails.initialRecordScoreTooltip',
+          {
+            defaultMessage:
+              'A normalized score between 0-100, which indicates the relative significance of the anomaly record result. This is the initial value that was calculated at the time the bucket was processed.',
```

Comment:
Maybe it can be further simplified to something like this?

```suggestion
              'A normalized score between 0-100, which indicates the relative significance of the anomaly record when the bucket was initially processed.',
```

Reply:
Done in https://github.com/elastic/kibana/pull/108216/commits/fbb27179f192a4dba74ff02620f046627b8cf8cf